### DISCUSSION TITLE
Not receiving email to login

### DISCUSSION THREAD

**Discussion by eduardotamaki** [vercel/community#3380] [Feature Question] [answered]
### Question

Hello Vercel Support Team,

I am currently experiencing difficulties accessing an old account of mine through the email login. Despite multiple attempts, I am still unable to gain entry, even though I have been receiving the requisite login emails.

I am wondering if this issue might be related to the domain associated with this account. Could you kindly assist me in verifying this, or guide me to possible solutions if the problem lies elsewhere?

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with login. Since we'll need to discuss personal account information, please email [registration@vercel.com](mailto:registration@vercel.com) to open a support case. That way we can verify your info safely out of this public post. 

Be sure to include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Reply by q3warserver** [reply]
Hi Amy. Since this discussion has been marked answered and is similar to my discussion (which has gone unanswered) I thought I would try replying here.

I was briefly in contact with the Vercel registration team via the email address that you provided, and I sent them information to help them identify my account. I even edited a txt record with my case number on my domain to prove I owned the account but no one is responding to me. My post in this forum has also gone unanswered. I am starting to think I have unknowingly done something wrong and I am intentionally being ignored. I am simply just trying to regain access to my account. I would appreciate it if someone would respond to me.
Thanks!
My discussion (https://github.com/orgs/vercel/discussions/3375)

**Comment by UG-SERVICE** [comment]
facing issue while login 
ug.trading.services@gmail.com
Error->We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.
